content/dessert/fresh-apple-cake.md

```diff
@@ -0,0 +1,75 @@
+---
+title:  Fresh Apple Cake"
+category: "dessert"
+date: 2024-02-06
+recipeYear: YYYY
+author: "Grandma (Bunny) Lawrence"
+prepTime: ""
+cookTime: "55-60m"
+vegetarian: true
+---
+
+### Ingredients
+
+#### Cake
+
+- 1 1/2 cups salad oil
+- 2 cups sugar
+- 2 large eggs
+- 1 teaspoon vanilla
+- 2 1/2 cups flour
+- 1 cup chopped pecans
+- 3 cups raw apples 
+- 1 teaspoon salt
+- 1 teaspoon baking soda
+- 2 teaspoons baking powder 
+
+#### Caramel Icing
+
+- 1 stick of margarine
+- 1/2 teaspoon of salt
+- 2 tablespoons evaporated milk
+- 1 cup firmly packed brown sugar
+- Icing sugar (added to help spreading consistency)
+
+---
+
+### Instructions
+
+Preheat oven to 350°F 
+
+#### Cake
+
+1. Peel and dice apples and set aside.
+2. Place oil in mixing bowl of a stand mixer.
+3. Add sugar and eggs.
+4. Beat until creamy at low speed.
+5. Add vanilla.
+6. In a separate bowl, sift flour with salt, baking soda and baking powder.
+7. Add small amount of flour to creamed mixture, beating well after each addition. 
+8. When all dry ingredients have been added, batter will be stiff.
+9. Remove beaters and fold in pecans and apples.
+10. Spread evenly in a large pan (9x11x12 inches)
+11. Bake for 55-60 minutes or until cake tester comes out clean. 
+12. Allow to cool before icing.
+
+#### Icing
+
+1. Place margarine, salt and brown sugar in saucepan. 
+2. Heat just enough to melt margarine and mix until sugar dissolves.
+3. Add just enough icing sugar to make enough consistency to spread. 
+
+### Comments
+
+Bunny Lawrence's comments on the recipe card:
+• very, very good Mother Lawrence
```

Comment:
These bullet characters need to be hyphens for Markdown to understand that they should be bullets.  Writing these as actual bullets probably looks _reasonably_ correct, but I don't think they'll get formatted correctly into a list?

```suggestion
- very, very good Mother Lawrence
```